cpp/cmake/common.cmake

```diff
@@ -3,6 +3,25 @@ cmake_minimum_required(VERSION 3.16)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if (WIN32 AND NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/packages)
+    message(STATUS "Downloading Ice NuGet package")
+    # option(ICE_NUGET_INSTALL "Use NuGet to install Ice during configuration" ON)
+    set(ICE_NUGET_SOURCE "https://download.zeroc.com/nexus/repository/nuget-nightly/" CACHE STRING "Ice NuGet package source")
+    # if (ICE_NUGET_INSTALL)
+    execute_process(
+        COMMAND nuget install -Source ${ICE_NUGET_SOURCE} -OutputDirectory ${CMAKE_CURRENT_BINARY_DIR}/packages zeroc.ice.v143 -Prerelease
```

Comment:
You can also pass `-x` to extract directly to zeroc.ice.v143 with including the package version.

Does install create the packages directory?

Reply:
yes